<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupProfileGUI</class>
 <widget class="QWidget" name="GroupProfileGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGraphicsView" name="groupPhoto"/>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="groupNameLabel">
         <property name="text">
          <string>Group Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="descriptionBrowser"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTableWidget" name="memberTable"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
